### ISSUE TITLE
Data to include (and where it's coming from)

### ISSUE THREAD

**Issue by limnoliver** [USGS-R/forecast-data-release-v2#1]
1. Spatial
- [ ] stream reaches on which we're forecasting
- [ ] reservoirs modeled with GLM

2. Model drivers 
- [ ] GEFS archive for forecasting
- [ ] NLDAS files for GLM
- [ ] modeled flow and temperature at ins/outs of reservoirs for GLM
- [ ] observed flow and temperature at ins/outs of reservoirs for GLM 

3. Model Config
- [ ] GLM config, (config for forecasts go into the code release) 

4. Predictions
- [ ] reservoir outlet temperatures from GLM
- [ ] DWALLIN - which is GLM + SNTemp weighted predictions of temperature along DRB network 
- [ ] PGDL-DA forecasts of water temperature (+ metadata)

5. Evaluation
- [ ] paired predictions and observations
- [ ] code to recreate figures/model error?

**Comment by jzwart** [comment]
>GEFS archive for forecasting - @jzwart will point us to Caldera location

Will be here `/caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/2_forecast_prep/in/met_drivers`. There are files there right now, but those are the files from last year (only 5 priority segments). Will ping when they're updated. 

>PGDL-DA forecasts of water temperature (+ metadata) - @jzwart will point us to Caldera location

Will be here `/caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/4_model_forecast/out`. There are files there right now, but those are from last year. Will ping when they're updated. There will be `.nc` files for the data and `.xml` files for the metadata associated with each `.nc` file 

>paired predictions and observations - @jzwart will point us to Caldera location

Will be here `/caldera/projects/usgs/water/iidd/datasci/water-prediction/evaluate-forecast/out/all_mods_with_obs.rds`. There is a file there right now, but it is from last year. Will ping when it's updated.

**Comment by jzwart** [comment]
GEFS archive for forecasting is ready to go into data release - files are here `/caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/2_forecast_prep/in/met_drivers` . There should 372 `.nc` files total

**Comment by msleckman** [comment]
GEFS archive dir `/met_drivers` added to `forecast-data-release-v2/in_data/03_driver/`. There are 372 nc files now there.

**Comment by msleckman** [comment]
>  Will be here /caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/4_model_forecast/out. There are files there right now, but those are from last year. Will ping when they're updated. There will be .nc files for the data and .xml files for the metadata associated with each .nc file

@jzwart will all the files (.nc + .xml) need to be transferred over and zipped in data release `in_data` folder?

**Comment by msleckman** [comment]
yup should the `.xml` and `.nc` be zipped up separately or together?

**Comment by limnoliver** [comment]
Last year, we zipped them up together. See metadata that described how they are stored here: https://github.com/limnoliver/forecast_data_release/blob/main/in_text/text_data_release.yml#L275-L279

**Comment by jzwart** [comment]
We'll have the output from the forecast pipeline will zip up all the forecast files for each day (so 1 zip file / day), and then we can zip all of those zip files together. The `.xml` and `.nc` files will be zipped together in the daily zip files, mirroring what we did last year

**Comment by msleckman** [comment]
> stream reaches on which we're forecasting (local build from delaware-model-prep but filtered based on model predictions)

@limnoliver I can incorporate this into the pipeline, but what do we filter do?

**Comment by msleckman** [comment]
> Will be here /caldera/projects/usgs/water/iidd/datasci/water-prediction/evaluate-forecast/out/all_mods_with_obs.rds. There is a file there right now, but it is from last year. Will ping when it's updated.

I' ve copied over all_mods_with_obs.rds to forecast-data-release-v2/in_folder in caldera but renamed it all_mods_with_obs_lst_yr.rds. @jzwart  is the file now updated? 

> PGDL-DA forecasts of water temperature (+ metadata) - @jzwart will point us to Caldera location

> Will be here /caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/4_model_forecast/out. There are files there right now, but those are from last year. Will ping when they're updated. There will be .nc files for the data and .xml files for the metadata associated with each .nc file

I create a PGDL-DA folder in the in_data folder. @jzwart just checking, can I copy over the `.nc` and `.xml` files in the `/caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/4_model_forecast/out`

**Comment by jzwart** [comment]
>I've copied over all_mods_with_obs.rds to forecast-data-release-v2/in_folder in caldera but renamed it all_mods_with_obs_lst_yr.rds. @jzwart is the file now updated?

It should be updated next Monday / Tuesday 

>I create a PGDL-DA folder in the in_data folder. @jzwart just checking, can I copy over the .nc and .xml files in the /caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da/4_model_forecast/out

Final model `.nc` and `.xml` files will be ready next Monday / Tuesday. @jdiaz4302  has some placeholder files if you need something sooner (but based on check-in meeting, sounds like next Monday / Tuesday will be ok).

**Comment by jdiaz4302** [comment]
In case they're helpful, the placeholder files can be found in `/caldera/projects/usgs/water/iidd/datasci/water-prediction/forecast-pgdl-da-jdiaz/4_model_forecast/out`

**Comment by msleckman** [comment]
> - [ ] reservoirs modeled with GLM (local build from delaware-model-prep)

@limnoliver  Is the source of this canonical_lakes_sf.rds as well?

**Comment by msleckman** [comment]
@limnoliver 

> Predictions
reservoir downstream predictions GLM, see pattern and location of files [here](https://github.com/USGS-R/drb-temp-data-release/blob/main/5_predictions.yml#L25); built locally and provided by Hayley.

> Model drivers
modeled flow and temperature at ins/outs of reservoirs for GLM; [data](https://github.com/USGS-R/delaware-model-prep/blob/main/9_collaborator_data.yml#L314), then [filtered further for release](https://github.com/USGS-R/drb-temp-data-release/blob/main/5_predictions.yml#L19)

Does res_io_sntemp.feather belong in driver or predictions?
